lib/autosign/validator.rb

```diff
@@ -66,52 +51,55 @@ def perform_validation(challenge_password, certname, raw_csr)
     # Do not override or use this class in child classes. This is the class that gets called
     # on validator objects.
     def validate(challenge_password, certname, raw_csr)
-      @log.debug "running validate"
-      fail unless challenge_password.is_a?(String)
-      fail unless certname.is_a?(String)
+      @log.debug "attempting to validate using #{name}"
```

Comment:
This cleans up some logging mechanisms here.  Previously they were inside a loop when they should be in this method here.  Because they were in both places we were seeing double logging for the same kind of message.